### PR TITLE
✨ Add get profile properties functions

### PR DESCRIPTION
**Pull request by weisslertal**
## Proposed Changes

Added new helper functions to allow easily fetching profile properties
The new functions are:  
`getName`
`getGivenName`
`getMobileNumber`
They are all similar to the existing `getEmail` function

## Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project
- [x] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed

### CODE REVIEW

**Review by jankoenig — APPROVED** [review]
Thank you! I updated the docs accordingly ✅

File: platforms/platform-alexa/src/AlexaUser.ts [hunk 1]
```diff
@@ -29,13 +29,29 @@ export class AlexaUser extends JovoUser<Alexa> {
...
+
+  async getName(): Promise<string | undefined> {
+    return await this.getProfilePropery(ProfileProperty.NAME) as string;
+  }
+
+  async getGivenName(): Promise<string | undefined> {
+    return await this.getProfilePropery(ProfileProperty.GIVEN_NAME) as string;
+  }
+
+  private async getProfilePropery(property: ProfileProperty): Promise<unknown> {
```
Comment (aswetlow) [inline platforms/platform-alexa/src/AlexaUser.ts]: Typo in `getProfilePropery`

**Comment by aswetlow** [comment]
Thank you for the PR @weisslertal. This is helpful.

We're ready to merge after the typo is fixed.

**Comment by weisslertal** [comment]
> Thank you for the PR @weisslertal. This is helpful.
> 
> We're ready to merge after the typo is fixed.

@aswetlow Typo is fixed:) Thanks!

**Comment by aswetlow** [comment]
Thank you, @weisslertal !

I just realized that we can get rid of the casting after modifying `getProfileProperty` with generics.

```typescript
private async getProfileProperty<PROPERTY extends ProfileProperty>(
    property: PROPERTY,
  ): Promise<CustomerProfileApiResponse<PROPERTY> | undefined> {
    const request: AlexaRequest = this.jovo.$request;
    return sendCustomerProfileApiRequest(
      property,
      request.getApiEndpoint(),
      request.getApiAccessToken(),
    );
  }
```

Results in 
```typescript
async getEmail(): Promise<string | undefined> {
    return this.getProfileProperty(ProfileProperty.EMAIL);
  }

  async getMobileNumber(): Promise<{ countryCode: string; mobileNumber: string } | undefined> {
    return this.getProfileProperty(ProfileProperty.MOBILE_NUMBER);
  }

  async getName(): Promise<string | undefined> {
    return this.getProfileProperty(ProfileProperty.NAME);
  }

  async getGivenName(): Promise<string | undefined> {
    return this.getProfileProperty(ProfileProperty.GIVEN_NAME);
  }
```

What do you think?

**Comment by weisslertal** [comment]
> 

@aswetlow Great suggestion! I pushed the changes

**Review by aswetlow — APPROVED** [review]
Thanks!

**Pull request merged**